spec/support/integration_test_runner.rb

```diff
@@ -28,13 +28,13 @@ def initialize(codeowners: [], owners: [], file_tree: {}, flags: [], answers: []
   end
 
   # rubocop: disable Lint/HandleExceptions
-  def run
+  def run(command: 'check', flags: @flags)
     setup_project
     $stdout = CaptureIO.new
     setup_io_listeners
     $stdout = STDOUT
     begin
-      Codeowners::Cli::Main.start(['check', PROJECT_PATH, *flags])
+      Codeowners::Cli::Main.start([command, PROJECT_PATH, *flags])
```

Comment:
👍

Reply:
Sorry, haven't commented on that, I was trying to make this less specific for the check command. I think we can improve a lot, but this was the minimum effort solution that I've found.

But I think you understand given your :+1: reaction.